test/models/metadata-fields/metadata-field.test.ts

```diff
@@ -0,0 +1,152 @@
+import { expect } from '@open-wc/testing';
+import {
+  FieldParserInterface,
+  FieldParserRawValue,
+} from '../../../src/models/metadata-fields/field-parser-interface';
+
+import { MetadataField } from '../../../src/models/metadata-fields/metadata-field';
+
+describe('Metadata Field', () => {
+  it('can be properly instantiated with single value', () => {
+    class MockParser implements FieldParserInterface<string> {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      parseValue(rawValue: any): string {
+        return rawValue;
+      }
+    }
+
+    const parser = new MockParser();
+    const metadataField = new MetadataField(parser, 'foo');
+
+    expect(metadataField.rawValue).to.equal('foo');
+    expect(metadataField.value).to.equal('foo');
+    expect(metadataField.values).to.deep.equal(['foo']);
+  });
+
+  it('can be properly instantiated with array value', () => {
+    class MockParser implements FieldParserInterface<string> {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      parseValue(rawValue: any): string {
+        return rawValue;
+      }
+    }
+
+    const parser = new MockParser();
+    const metadataField = new MetadataField(parser, ['foo', 'bar', 'baz']);
+
+    expect(metadataField.rawValue).to.deep.equal(['foo', 'bar', 'baz']);
+    expect(metadataField.value).to.equal('foo');
```

Comment:
When you access `.value`, it just returns the first item in the `values` array, whose order is determined by the metadata response. It's just a convenience method since a lot of fields only ever have a single value or we're only interested in the first value.